jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -39,7 +39,7 @@
           (ros::set-param (format nil "~a_hand/state" (send *ri* :arm-symbol2str arm)) (symbol2str state)))
         (:initialize
           (ros::ros-info "[main] ~a, ~a" arm state)
-          (setq tote-contents (send *ri* :get-tote-contents))
+          (setq tote-contents (get-tote-contents))
```

Comment:
I think what we want to do with `get-tote-contents` is specific to APC, but not specific to Baxter.
So how about making new `apc-util.l` and put `get-tote-contents` in it?